### DISCUSSION TITLE
ExternalK8sLabelTag 和 processor_drop 不生效问题

### DISCUSSION THREAD

**Discussion by hanzoi** [alibaba/ilogtail#493] [Help] [answered]
配置如下：
<img width="298" alt="image" src="https://user-images.githubusercontent.com/61114924/198232099-6a3a81ff-a077-4553-a2cf-632f9551cfe0.png">

问题一：尝试 drop remote_addr 是成功的，drop tags里的字段都是无效的，尝试直接 drop 原始字段，也无法 drop 掉 tags 里的字段

问题二：ExternalK8sLabelTag 使用此参数为 tags 增加字段，但是未生效

k8s ds方式部署，使用 flusher_kafka_v2 ，从kafka里拿到的日志数据 没有添加 service 字段，也没有 drop 掉相关的字段
<img width="947" alt="企业微信截图_cf60411e-614f-491e-9754-d7f234b470de" src="https://user-images.githubusercontent.com/61114924/198232820-a4538b4c-a45a-4cf4-9055-738325c2454a.png">

**Comment by hanzoi** [comment, marked as answer]
已解决

**Reply by messixukejia** [reply]
怎么解决建议补充描述下，后续其他人可以做个参考
